documentation/api.tex

```diff
@@ -0,0 +1,69 @@
+\chapter{API documentation}
+This chapter will cover the documentation of the back-end API.
+
+\section{GraphiQL}
```

Comment:
Yeah, we could do that. Maybe in a section about the tooling we use, if we do that.